tests/features_test.py

```diff
@@ -0,0 +1,16 @@
+import pandas as pd
+from nxontology.examples import create_metal_nxo
+
+from nxontology_ml.features import NxontologyFeatures
+from tests.utils import read_test_resource
+
+
+def test_NxontologyFeatures() -> None:
+    # FIXME: Overly high level test
+    test_nxo = create_metal_nxo()
+    test_nxo_features: NxontologyFeatures[str] = NxontologyFeatures(test_nxo.graph)
+    test_df = test_nxo_features.get_features_df()
+    expected_df = pd.read_json(read_test_resource("metal_nxo_features.json"))
```

Comment:
How do we regenerate these test files? I imagine they will be changing a lot.

Reply:
For these kind of non-regression oriented unit tests, I usually run the test in debug mode, place a break point and dump the values of the variables I want to test in the debugger. 
In this case I ran `test_df.to_json()` in PyCharm's debugger:
<img width="573" alt="image" src="https://github.com/related-sciences/nxontology-ml/assets/1596570/d5a7b38e-403a-4773-a8cd-03853e522267">